src/renault_api/kamereon/__init__.py

```diff
@@ -311,17 +320,28 @@ async def set_vehicle_action(
     vin: str,
     endpoint: str,
     attributes: Dict[str, Any],
+    actions: str = "actions",
     endpoint_version: Optional[int] = None,
     data_type: Optional[Dict[str, Any]] = None,
 ) -> models.KamereonVehicleDataResponse:
-    """POST to /v{endpoint_version}/cars/{vin}/actions/{endpoint}."""
+    """POST to /v{endpoint_version}/cars/{vin}/{actions}/{endpoint}."""
     car_adapter_url = get_car_adapter_url(
         root_url=root_url,
         account_id=account_id,
         version=endpoint_version or _get_endpoint_version(ACTION_ENDPOINTS[endpoint]),
         vin=vin,
     )
-    url = f"{car_adapter_url}/actions/{endpoint}"
+
+    if actions == "charge":
```

Comment:
I think it would be better to use a separate (new) argument to decide on the car_adapter_url:
```python
    if car_adapter_type == "kcm":
        car_adapter_url = get_kcm_car_adapter_url(...
    else:
        car_adapter_url = get_car_adapter_url(...
```